### PR TITLE
Display test results in console.

### PR DESCRIPTION
**Pull request by danirueda**
It's only a small addition but I'll still looking for news of them.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -64,3 +64,10 @@ eclipse {
     }
 }
 
+test {
+	//we want display the following test events
```
Comment (RecuencoJones) [inline build.gradle]: use 4 spaces instead of tabs

File: description.md [hunk 2]
```diff
@@ -59,6 +59,7 @@ In addition to the commands said before, there are other interesting commands:
 - **test:** Runs the unit tests.
 - **clean:** Deletes the build directory.
 - **jar:** Assembles a jar archive containing the main classes.
+- **clean test:** [Displays the test results in console.](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/issues/42)
```
Comment (RecuencoJones) [inline description.md]: is it `clean test` or just `test`? If it works for `test` task, update it in line 59 and remove this one

**Comment by RecuencoJones** [comment]
Also, @danirueda, please fix the conflicts by merging this repo's master branch with yours

**Comment by adrianblade** [comment]
@fjlopez you can merge this branch.
@RecuencoJones

**Comment by danirueda** [comment]
I have corrected the errors. Do I must make a new pull request?? Or Is this still having validity???

**Comment by SAReyes** [comment]
Hi @danirueda, any pull request can be updated at anytime, there's no need for it to be closed unless the thread becomes unreadable or a reviewer decides it should be closed. That being said, you've uploaded merge conflicts, please solve them.

 Moreover, I noticed you've removed the `clean` goal, given that it is required for the test results to be shown in the console output we are expecting an explanation for it.

**Comment by danirueda** [comment]
@SAReyes I don't know how I can resolve the merge conflicts, I'm a noob in the branch merging. I have done what @fjlopez said to me:
`$ git fetch upstream
$ git checkout master
$ git merge upstream/master`

Another thing, can github make live notifications like facebook? It's tedious reciving e-mails all time.

**Comment by SAReyes** [comment]
[This](https://githowto.com/resolving_conflicts) would be a starting point, you can edit the file manually, if you take a look at the files you've modified you may find some with this pattern `<<<<<<< HEAD` {{your changes}} `=======` {{remote changes}} `>>>>>>> upstream/master`, you will need to decide which changes to keep, or even both, then you only need to make a new commit and push it to your forked repository.

#45 is open to explain how to do this alongside some useful tools.

Regarding notifications, I don't know if you can get desktop notifications, you've got a notifications page though, it is located at the right side of the navbar. Most devs use slack with its github integration for this purpose, it is worth a look.

**Comment by RecuencoJones** [comment]
@SAReyes maybe it would be a good idea to create a Slack team for those that will be participating in the subject. @fjlopez what do you think?

File: description.md [hunk 3]
```diff
@@ -58,6 +58,32 @@ Other possibility related to `gradle build` are `gradle assemble` that compiles
 
 Once this is done, you can start the application using the command `gradle bootrun`. If everything goes fine, the application will have been compiled and started Spring Framework with the application. Now, you can access from your favourite browser by typing in the address bar: *localhost:8080*.
 
+It can do a testLoggin in the console. This resolves issue #42.
```
Comment (RecuencoJones) [inline description.md]: add link for issue #42

**Comment by fjlopez** [comment]
If you run `gradle test`, the code is unmodified and a previous test has been run then the output will be:
```
:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:compileTestJava UP-TO-DATE
:processTestResources UP-TO-DATE
:testClasses UP-TO-DATE
:test UP-TO-DATE

BUILD SUCCESSFUL
````

You must force a clean before running tests in such case.

**Pull request merged**